types/es-abstract/GetIntrinsic.d.ts

```diff
@@ -22,11 +22,398 @@ declare function GetIntrinsic<K extends keyof GetIntrinsic.Intrinsics>(
 declare function GetIntrinsic(name: string, allowMissing?: boolean): unknown;
 export = GetIntrinsic;
 
+type numeric = number | bigint;
+interface TypedArray<T extends numeric = numeric> extends Readonly<ArrayBufferView> {
+    /** The length of the array. */
+    readonly length: number;
+    [index: number]: T;
+}
```

Comment:
this isn't a builtin type??

Reply:
Not in **TypeScript**, that only has `ArrayBufferView` and the individual classes, but `%TypedArray%` and `%TypedArray.prototype%` are untyped.